addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -241,6 +241,10 @@ function factory(dependencies) {
                 channel.mass_mailing &&
                 this.env.session.notification_type === 'email'
             ) {
+                this._handleNotificationChannelSeen(channelId, {
+                    last_message_id: messageData.id,
+                    partner_id: this.env.messaging.currentPartner.id,
+                });
```

Comment:
This fix will not work when data are loaded at init, and not received from the bus.

Also the handler should never be called manually. There should have actual business methods to make a channel as seen if necessary.

Reply:
It was tested and nothing was shown as not working. What is exactly your failing use case ?

Reply:
Just have multiple users on a mailing channel, post on the channel as one user (while the other user does not have the client open), then open the client with the other user and notice the channel is not marked as seen.

Reply:
This is purely for mailing channel, so in v15 it shouldn't even have "channel seen" feature on them in the first place.

Reply:
Ok then technical limitations. Mailing channels is already a niche feature, people could sometimes have a notification where they should not, tant pis.